modules/data/tools/smart_recorder/proto/smart_recorder_status.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto2";
+
+package apollo.data;
+
+import "modules/common/proto/header.proto";
+
+enum RecordingState {
+  RECORDING = 0;
+  TERMINATING = 1;
+  STOPPED = 2;
```

Comment:
In proto enum, the first value will be the default if the field is not set.
Maybe better to put `STOPPED` as the default or something like `UNKNOWN`.